include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h

```diff
@@ -1221,33 +1221,9 @@ namespace internal
       }
 
       // ------------- prolongation matrix (0) -> identity matrix --------------
-      {
-        AssertDimension(fe_fine.n_base_elements(), 1);
-        if (reference_cell == ReferenceCells::get_hypercube<dim>())
-          {
-            const auto fe = create_1D_fe(fe_fine.base_element(0));
-
-            transfer.schemes[0].prolongation_matrix_1d.resize(
-              fe->dofs_per_cell * fe->dofs_per_cell);
 
-            for (unsigned int i = 0; i < fe->dofs_per_cell; i++)
-              transfer.schemes[0]
-                .prolongation_matrix_1d[i + i * fe->dofs_per_cell] =
-                Number(1.0);
-          }
-        else
-          {
-            const unsigned int n_dofs_per_cell =
-              fe_fine.base_element(0).n_dofs_per_cell();
-
-            transfer.schemes[0].prolongation_matrix.resize(n_dofs_per_cell *
-                                                           n_dofs_per_cell);
-
-            for (unsigned int i = 0; i < n_dofs_per_cell; i++)
-              transfer.schemes[0].prolongation_matrix[i + i * n_dofs_per_cell] =
-                Number(1.0);
-          }
-      }
+      // nothing to do since the prolongation matrices are not used a short-cut
+      // code path is used during prolongation/restriction
```

Comment:
Do you want to say the following:
```suggestion
      // nothing to do since for empty prolongation matrices a short-cut
      // code path is used during prolongation/restriction
```

Reply:
I changed to the following:

https://github.com/dealii/dealii/blob/6cc7ab6c32d01931af19f2aa44da4005990af32f/include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h#L1225-L1226